style/preview.less

```diff
@@ -215,6 +215,55 @@
 		}
 	}
 
+	&-error {
```

Comment:
remember to center the message (both horizontal and vertical), do check on the larget screen (ex. iPad size)